README.md

```diff
@@ -209,7 +209,10 @@ Once deployed, check out the API docs with the following links:
 - https://$YOUR_DNS_DOMAIN/jobs-service/api/v1/docs
 - https://$YOUR_DNS_DOMAIN/llm-service/api/v1/docs
 
-> Alternatively you can link with IP address e.g. http://x.x.x.x/authentication/api/v1/docs to verify API endpoints.
+> Alternatively, you may use the IP address e.g. `http://$BASE_IP_ADDRESS/authentication/api/v1/docs` to verify API endpoints
+```
+BASE_IP_ADDRESS=gcloud compute addresses list --global --format="value(address)"
```

Comment:
Don't you need a bracket to wrap the command line? 
```suggestion
BASE_IP_ADDRESS=$(gcloud compute addresses list --global --format="value(address)")
```

Reply:
Thank you for catching 👍